__tests__/index.test.js

```diff
@@ -113,6 +113,40 @@ describe('/api/articles/:id', () => {
   })
 })
 
+describe('/api/articles/:id/vote_count', () => {
+  it('PATCH:200 issues a response with a database-updated article, where `.vote_count` is incremented by increment', async () => {
+    const testArticle = testArticles[0]
+
+    const reqBody = { vote_increment: 3 }
+
+    const res = await request(app)
+      .patch('/api/articles/1/vote_count')
+      .send(reqBody)
+      .expect(200)
+
+    const { article } = res.body
+
+    expect(article).toEqual({
+      id: 1,
+      ...testArticle,
+      created_at: article.created_at, // no way to avoid
+      vote_count: testArticle.vote_count + reqBody.vote_increment,
+    })
+  })
+
+  it.only('PATCH:400 (insufficient request body) issues a response with `Bad Request` error text', async () => {
+    const reqBody = {}
+
+    const res = await request(app)
+      .patch('/api/articles/1/vote_count')
+      .send(reqBody)
+      .expect(400)
+
+    const errText = res.error.text
+    expect(errText).toBe('Bad Request')
+  })
+})
+
```

Comment:
Please add the following tests :) 
-Invalid id (400)
-Non-existant id (404)
-vote_increment not an integer (400)